ztools/ztls/ztls_names.go

```diff
@@ -435,6 +436,21 @@ func init() {
 	pointFormatNames[0] = "uncompressed"
 	pointFormatNames[1] = "ansiX962_compressed_prime"
 	pointFormatNames[2] = "ansiX962_compressed_char2"
+
+	// https://tools.ietf.org/html/draft-ietf-tls-tls13-18#section-4.2.3
```

Comment:
Can we make the integers themselves named constants, then define the names map in terms of the constants? In the past, I've just autogenerated this code using awk, then copy/pasted it.